forgerock-openbanking-uk-aspsp-common/src/main/java/com/forgerock/openbanking/common/model/onboarding/ManualRegistrationRequest.java

```diff
@@ -20,16 +20,8 @@
  */
 package com.forgerock.openbanking.common.model.onboarding;
 
-import lombok.*;
-
 import java.util.List;
 
-@Data
```

Comment:
Why remove lombok? you can still reduce the noise with lombok and implement equals and hashcode manually if thats what is required.

Reply:
Not sure why that's showing like that. It's in a different project, not a part of openbanking-aspsp at all?